source/change-streams/tests/unified/change-streams.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
This file was generated from `change-streams.yaml` via [`js-yaml`](https://github.com/nodeca/js-yaml).